docs/api/ReactWrapper/simulate.md

```diff
@@ -71,12 +71,12 @@ const wrapper = mount((
   />
 ));
 
-expect(wrapper.find('input').at(0).prop('value')).toEqual(10);
-expect(wrapper.find('input').at(1).prop('value')).toEqual(20);
+expect(wrapper.find('input').at(0).props().value).to.equal(10);
+expect(wrapper.find('input').at(1).props().value).to.equal(20);
```

Comment:
the change from `toEqual` to `.to.equal` is good; i'm not sure about whether it's better to use `.props().x` or `.prop('x')` here. is there a reason you made the choice?

Reply:
The reason I made the choice is `.props()` return the object, and `.props().x` return the value.

`.props("x")` return `{ name: 'width', value: 10, onChange: [Function: onChange] }`.
`.prop().x`  return 10